### PR TITLE
Feature/write the docs workflow

### PR DESCRIPTION
**Pull request by matilote**

### CODE REVIEW

File: src/Nethermind/Nethermind.WriteTheDocs/CliDocsGenerator.cs [hunk 1]
```diff
@@ -85,7 +85,7 @@ public void Generate()
 
             Console.WriteLine(result);
             File.WriteAllText("cli.rst", result);
-            File.WriteAllText("../../../../../../docs/source/cli.rst", result);
```
Comment (tkstanczak) [inline src/Nethermind/Nethermind.WriteTheDocs/CliDocsGenerator.cs]: why did you need to change the folder structure? will it still work when invoked manually?

File: src/Nethermind/Nethermind.WriteTheDocs/configuration.rst [hunk 2]
```diff
@@ -423,11 +427,11 @@ SyncConfig
    default value: true
 
  DownloadReceiptsInFastSync
-   If set to 'true' then the receipts will be downloaded in the Fast Sync mode.
+   If set to 'true' then the receipts will be downloaded in the Fast Sync mode. This will slow down the process by a few hours but will allow you to interact with dApps that execute extensive historical logs searches (like Maker CDPs).
    default value: true
 
  FastBlocks
-   If set to 'true' then in the Fast Sync mode blocks will be first downloaded from the provided PivotNumber downwards.
+   If set to 'true' then in the Fast Sync mode blocks will be first downloaded from the provided PivotNumber downwards. This allows for parallelization of requests with many sync peers and with no need to worry about syncing a valid branch (syncing downwards to 0). You need to enter the pivot block number, hash and total difficulty from a trusted source (you can use etherscan and confirm with other sources if you wan to change it).
```
Comment (tkstanczak) [inline src/Nethermind/Nethermind.WriteTheDocs/configuration.rst]: this has already been regenerated on master - did you branch from an old version or are you generting to a wrong folder? (local folder instead of actual output?)

File: src/Nethermind/Nethermind.WriteTheDocs/CliDocsGenerator.cs (hunk 1 above)
Comment (matilote) [reply src/Nethermind/Nethermind.WriteTheDocs/CliDocsGenerator.cs]: I was getting System.IO errors while trying to run `dotnet run Nethermind.WriteTheDocs` manually. This fix removed those errors.
